deviceModeInit.js

```diff
@@ -177,11 +177,26 @@ var rudderTracking = (function () {
     };
     if (pages[path] === "Registration Viewed") {
       rudderanalytics.track(pages[path], properties);
+      rs$('form[action="/account"] [type="submit"]').on(
+        "click",
+        userRegistered
+      );
     } else {
       rudderanalytics.page(category, pageName, properties);
     }
   }
 
+  function userRegistered() {
+    const userEmail = rs$('form[action="/account"] [type="email"]').val();
+    const firstName = rs$('form[action="/account"] [name="customer[first_name]"]').val();
+    const lastName = rs$('form[action="/account"] [name="customer[last_name]"]').val();
+    rudderanalytics.identify(__st.cid.toString(),{ 
```

Comment:
`cid` may not be immediately available after submit check once

Reply:
On investigating further, it is seen that:
After `submit` is clicked, the stores redirect to their homepage, where in `init()` gets called and `userId` gets associated with `cid`, which is now available post signup.